### PR TITLE
Add Spruce

### PR DESCRIPTION
**Pull request by littledivy**
Spruce is a tiny but powerful open-source social networking platform made with Node.js and MongoDB.

Thank You!

### CODE REVIEW

File: _data/projects/Spruce.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Spruce
+desc: An open source social media platform built with Node.js and MongoDB
+site: https://github.com/DivySrivastava/spruce
+tags:
+- javascript
+- nodejs
+- mongodb
+upforgrabs:
+  name: beginner
```
Comment (ritwik12) [inline _data/projects/Spruce.yml]: @DivySrivastava  Here, you should have the name for the label that you want to use to label your issues, such as "good first issue", "help wanted", "up-for-grabs", etc.

File: _data/projects/Spruce.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  link: https://github.com/DivySrivastava/spruce
```
Comment (ritwik12) [inline _data/projects/Spruce.yml]: Give the link to your label that you want to use, for example - https://github.com/DivySrivastava/spruce/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@DivySrivastava Also, tag your issues with the label that you have defined.

**Comment by littledivy** [comment]
Done! Thanks for the suggestions

File: _data/projects/Spruce.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/Spruce.yml]: @DivySrivastava you haven't yet changed name for the label. I can see that you've used good first issue label link. if you are using that then give same name here also

**Comment by ritwik12** [comment]
@DivySrivastava Please make changes

**Comment by littledivy** [comment]
Done

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@DivySrivastava Thanks!!

**Pull request merged**